lib/translator/assertions.rb

```diff
@@ -0,0 +1,80 @@
+module Translator
+  module Assertions
+    extend ActiveSupport::Concern
```

Comment:
why not make this part of our toolchain?
it's now being copied over many projects.